homeassistant/components/template/weather.py

```diff
@@ -183,32 +195,79 @@ def temperature_unit(self):
     @property
     def humidity(self):
         """Return the humidity."""
+        # unit from template: %
         return self._humidity
 
     @property
     def wind_speed(self):
         """Return the wind speed."""
-        return self._wind_speed
+        # unit from template: m/s
```

Comment:
The weather platform speed requires km/hr or mi/hr.  If this template indeed only accepts the metric variant, I would suggest asking for km/hr.

Reply:
I had two reasons to go for m/s: Neutrality :-) (not preferring km/h or mph) and the fact that I think that many weather stations report wind speed in m/s. But your comment makes sense - will change that...